src/main/java/pl/crm/domain/InvoiceEntry.java

```diff
@@ -0,0 +1,29 @@
+package pl.crm.domain;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
```

Comment:
Jak możesz to dodaj @AllArgsConstructor, @NoArgs... oraz @Data, to się przyda potem :D